packages/earth-map/src/components/filter-by/component.tsx

```diff
@@ -43,15 +43,15 @@ const FilterBy = (props: IProps) => {
         ? filterGroup.filter((x) => x !== value)
         : [...filterGroup, value],
     };
-    setPlacesSearch({
+    onChange({
       filters: cleanFilters({
         ...filters,
         ...newFilters,
       }),
     });
   };
 
-  const clearCheckedFilters = () => setPlacesSearch({
+  const clearCheckedFilters = () => onChange({
     filters: {}
   });
```

Comment:
I get this error, pls add key

Warning: Each child in a list should have a unique "key" prop.

Check the render method of `FilterBy`. See https://fb.me/react-warning-keys for more information.